src/Stripe/Infrastructure/Requestor.cs

```diff
@@ -48,36 +51,62 @@ public static string PostStringBearer(string url, StripeRequestOptions requestOp
 
             return ExecuteRequest(wr);
         }
+        
+        private static string ExecuteRequest(HttpRequestMessage requestMessage)
+        {
+            var response = HttpClient.SendAsync(requestMessage).Result;
```

Comment:
@jaymedavis This looks like deadlock city, see http://blog.stephencleary.com/2012/07/dont-block-on-async-code.html